Sparkle/SUBasicUpdateDriver.m

```diff
@@ -191,6 +191,11 @@ - (void)downloadUpdate
 {
     NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[self.updateItem fileURL]];
     [request setValue:[self.updater userAgentString] forHTTPHeaderField:@"User-Agent"];
+    if ([[self.updater delegate] respondsToSelector:@selector(updater:willDownloadUpdate:withRequest:)]) {
+        [[self.updater delegate] updater:self.updater
```

Comment:
Yes.  There were two prevailing conventions within the file and this was the one I used (the other was to have a local variable for the delegate).